ci-operator/config/openshift/multiarch-tuning-operator/openshift-multiarch-tuning-operator-v0.9.yaml

```diff
@@ -0,0 +1,245 @@
+      dependencies:
+      - env: OPERATOR_IMAGE
+        name: pipeline:multiarch-tuning-operator
+      - env: OO_BUNDLE
+        name: pipeline:multiarch-tuning-operator-bundle
+      from: operator-sdk-src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: ipi-gcp
+- as: e2e-gcp-multi-operator
+  cluster: build10
+  optional: true
+  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Za-z]+\.md$)|((^|/)\.github/)|((^|/)\.tekton/)|((^|/).*konflux\.Dockerfile$)
+  steps:
+    cluster_profile: gcp
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-initial
+      OPERATOR_IMAGE: pipeline:multiarch-tuning-operator
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
+      ADDITIONAL_WORKER_VM_TYPE: n2-standard-4
+      ADDITIONAL_WORKERS: "1"
+      COMPUTE_NODE_REPLICAS: "2"
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      OCP_ARCH: arm64
+    test:
+    - ref: ipi-install-heterogeneous
+    - as: test
+      cli: latest
+      commands: ./hack/deploy-and-e2e.sh
+      dependencies:
+      - env: OPERATOR_IMAGE
+        name: pipeline:multiarch-tuning-operator
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: ipi-gcp
+- as: e2e-aws-post-multi-operator-olm
+  cluster: build10
+  postsubmit: true
+  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Za-z]+\.md$)|((^|/)\.github/)|((^|/)\.tekton/)|((^|/).*konflux\.Dockerfile$)
+  steps:
+    cluster_profile: gcp
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-initial
+      OPERATOR_IMAGE: pipeline:multiarch-tuning-operator
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
+      ADDITIONAL_WORKER_VM_TYPE: n2-standard-4
+      ADDITIONAL_WORKERS: "1"
+      COMPUTE_NODE_REPLICAS: "2"
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      OCP_ARCH: arm64
+    test:
+    - ref: ipi-install-heterogeneous
+    - as: test
+      cli: latest
+      commands: USE_OLM=true ./hack/deploy-and-e2e.sh
+      dependencies:
+      - env: OPERATOR_IMAGE
+        name: pipeline:multiarch-tuning-operator
+      - env: OO_BUNDLE
+        name: pipeline:multiarch-tuning-operator-bundle
+      from: operator-sdk-src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: ipi-aws
+- as: security
+  cluster: build10
+  optional: true
+  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Za-z]+\.md$)|((^|/)\.github/)|((^|/)\.tekton/)|((^|/).*konflux\.Dockerfile$)
+  steps:
+    env:
+      PROJECT_NAME: multiarch-tuning-operator
+    workflow: openshift-ci-security
+- as: bundle
+  cluster: build10
+  commands: unset VERSION; NO_DOCKER=1 make bundle-verify
+  container:
+    from: operator-sdk-src
+  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Za-z]+\.md$)|((^|/)\.github/)|((^|/)\.tekton/)
+zz_generated_metadata:
+  branch: v0.9
```

Comment:
This one should not be main right?

Reply:
no and that one will be updated automatically. 

You'd run `make update` to generate the other files